layouts/index.html

```diff
@@ -0,0 +1,157 @@
+                                <em>Not currently running - check back 2021W2.</em>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card w-100">
+                        <img
+                            class="card-img-top img-fluid"
+                            src="https://d33wubrfki0l68.cloudfront.net/e2ad15d073dbd9d0a734261924994fc7fac2f302/bc5f9/files/classroom.jpg"
+                            alt=""
+                            width="348"
+                            height="196"
+                        >
+                        <div class="card-body">
+                            <h5 class="card-title">
+                                <a class="card-link" href="/services/courses/">Courses Database</a>
+                            </h5>
+                            <p class="card-text">We provide a database of student-sourced course reviews and descriptions to help you choose you next classes.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card w-100">
+                        <img
+                            class="card-img-top img-fluid"
+                            src="https://d33wubrfki0l68.cloudfront.net/e71f6e23e6c2a230223c6e13f9eb1ab25244b27e/00822/files/chips.jpg"
+                            alt=""
+                            width="348"
+                            height="196"
+                        >
+                        <div class="card-body">
+                            <h5 class="card-title">
+                                <a class="card-link" href="/cube/menu/">Cube Menu</a>
+                            </h5>
+                            <p class="card-text">We provide cheap snacks, like chips, pizza, candy, and drinks, in the Cube.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row justify-content-center">
+                <div class="col-lg-8 text-center large-text pb-5">
+                    <h1>Calendar</h1>
+                </div>
+            </div>
+            <iframe
+                id="open-web-calendar"
+                style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+                src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fubccsss.org%2Fevents%2Findex.ics&amp;title=UBC%20CSSS%20Calendar"
+                sandbox="allow-scripts allow-same-origin allow-top-navigation"
+                allowTransparency="true"
+                scrolling="no"
+                frameborder="0"
+                height="600px"
+                width="100%"
+                class="px-5 pb-5"
+            ></iframe>
```

Comment:
Could we also center the calendar + pull it out of the services + `<aside>` row? I think moving the iframe inside the `col` on line 137, and then moving the `row` on line 136 down outside of the parent row would work!

Reply:
I don't agree with this. I feel like we should keep sidebar spacing on the right since the Calander is not big enough to warrant the entire row. There's a lot of empty space on the sides.
![screencapture-localhost-1313-2022-08-29-23_18_19 (1)](https://user-images.githubusercontent.com/39626451/187364300-14b1b6ad-2d31-415a-9e85-9477c9083a08.png)

Reply:
Good point - I was initially thrown off a bit by the spacing (since it's not obvious the sidebar continues down) but we have this type of layout elsewhere in our pages so this's alright!